docs/requirements.txt

```diff
@@ -21,110 +20,103 @@ cloudpickle==1.6.0
 commonmark==0.9.1
-coverage==5.2.1
+coverage==5.3
 cycler==0.10.0
 cython==0.29.21
 dash==1.12.0
 dash-core-components==1.10.0
 dash-html-components==1.0.3
 dash-renderer==1.4.1
 dash-table==4.7.0
-dask==2.25.0
-decorator==4.4.2; python_version >= "3.3"
+dask==2.26.0
+decorator==4.4.2
 defusedxml==0.6.0
 deprecated==1.2.10
-descartes==1.1.0
-distributed==2.25.0
+distributed==2.26.0
 docutils==0.16
 entrypoints==0.3
 flake8==3.8.3
 flask==1.1.2
 flask-compress==1.5.0
-fsspec==0.8.0
+fsspec==0.8.2
 future==0.18.2
-gast==0.3.3
 google-api-core==1.22.2
 google-auth==1.21.1
-google-auth-oauthlib==0.4.1
 google-cloud-core==1.4.1
 google-cloud-storage==1.28.1
-google-pasta==0.2.0
 google-resumable-media==0.5.1
 googleapis-common-protos==1.52.0
-grpcio==1.31.0
-h5py==2.10.0
 heapdict==1.0.1
 idna==2.10
 imagesize==1.2.0
 importlib-metadata==1.7.0; python_version < "3.8"
+iniconfig==1.0.1
 ipykernel==5.3.4
 ipython==7.18.1
 ipython-genutils==0.2.0
 ipywidgets==7.5.1
 isort==4.3.21
 itsdangerous==1.1.0
-jedi==0.17.2; python_version >= "3.3"
+jedi==0.17.2
 jinja2==2.11.2
-joblib==0.16.0
+joblib==1.0.1
 jsonschema==3.2.0
 jupyter==1.0.0
 jupyter-client==6.1.7
 jupyter-console==6.2.0
 jupyter-core==4.6.3
-keras-preprocessing==1.1.2
-kfp==1.0.0
-kfp-server-api==1.0.0
+jupyterlab-pygments==0.1.1
+kfp==1.0.1
+kfp-server-api==1.0.1
 kiwisolver==1.2.0
 kornia==0.1.4.post2
 kubernetes==11.0.0
 locket==0.2.0
-markdown==3.2.2
 markupsafe==1.1.1
 matplotlib==3.3.1
 mccabe==0.6.1
 mistune==0.8.4
 more-itertools==8.5.0
 msgpack==1.0.0
-nbconvert==5.6.1
+nbclient==0.5.0
+nbconvert==6.0.2
 nbformat==5.0.7
+nest-asyncio==1.4.0
 notebook==6.1.4
 numpy==1.17.5
-nuscenes-devkit==1.0.2
 oauthlib==3.1.0
-opencv-python==4.4.0.42
-opt-einsum==3.3.0
+opencv-python==4.5.1.48
 packaging==20.4
 pandas==1.1.2
 pandocfilters==1.4.2
-parso==0.7.1; python_version >= "3.3"
+parso==0.7.1
 partd==1.1.0
 pathspec==0.8.0
 pbr==5.5.0
-pexpect==4.8.0; python_version >= "3.3" and sys_platform != "win32"
-pickleshare==0.7.5; python_version >= "3.3"
+pexpect==4.8.0; python_version >= "3.3" and sys_platform != "win32" or sys_platform != "win32"
+pickleshare==0.7.5
 pillow==7.2.0
-plotly==4.9.0
+plotly==4.10.0
 pluggy==0.13.1
 prometheus-client==0.8.0
 prompt-toolkit==3.0.7
 protobuf==3.13.0
 psutil==5.7.2
-ptyprocess==0.6.0; python_version >= "3.3" and sys_platform != "win32" or os_name != "nt"
+ptyprocess==0.6.0; python_version >= "3.3" and sys_platform != "win32" or sys_platform != "win32" or os_name != "nt" or python_version >= "3.3" and sys_platform != "win32" and (python_version >= "3.3" and sys_platform != "win32" or sys_platform != "win32")
 py==1.9.0
 pyasn1==0.4.8
 pyasn1-modules==0.2.8
 pycocotools==2.0.2
 pycodestyle==2.6.0
 pycparser==2.20
 pyflakes==2.2.0
-pygments==2.6.1
+pygments==2.7.0
 pyparsing==2.4.7
 pyquaternion==0.9.5
-pyrsistent==0.17.0
-pytest==5.4.3
+pyrsistent==0.17.3
+pytest==6.0.2
```

Comment:
Is there any reason why we upgrade those packages?

Reply:
Its not not manual update, after removal of poetry dependencies I updated requirement.txt using this: "poetry export --dev --without-hashes -f requirements.txt > docs/requirements.txt"